src/description.jl

```diff
@@ -61,8 +61,17 @@ end
 
 prefix(::Type{Year}) = "Y"
 prefix(::Type{Month}) = "Mo"
+prefix(::Type{Week}) = "W"
 prefix(::Type{Day}) = "D"
 prefix(::Type{Hour}) = "H"
 prefix(::Type{Minute}) = "M"
 prefix(::Type{Second}) = "S"
 prefix(::Type{Millisecond}) = "ms"
+
+# `Quarter` defined after Julia1.6.
+if VERSION >= v"1.6"
+    prefix(::Type{Quarter}) = "Q"
+end
+
+# informative error if no prefix is defined for the given type.
+prefix(T::Type{<:Period}) = error("A prefix for period $T has not yet been defined")
```

Comment:
If we'd like it to "just work" instead of throwing,
```suggestion

# Full name if no prefix is defined.
prefix(T::Type{<:Period}) = string(T)
```